video_xblock/tests/test_fields.py

```diff
@@ -0,0 +1,90 @@
+"""Tests for classes defined in fields.py."""
+
+import datetime
+import unittest
+
+from video_xblock.fields import RelativeTime
+
+
+class RelativeTimeTest(unittest.TestCase):
+    """
+    Test RelativeTime field.
+    """
+    delta = RelativeTime()
```

Comment:
Ah, I see now.
I think it's better to rename it to `time_delta`, because `delta` is a bit out of the blue.